mb-solrquerywriter/src/main/java/org/musicbrainz/search/solrwriter/moxy/StringIsrcAdapter.java

```diff
@@ -1,41 +0,0 @@
-/* Copyright (c) 2018 Sambhav Kothari
```

Comment:
This should be MetaBrainz, since you're doing this for work, not GSoC.

Reply:
Yup, the file is deleted? I'll update the rest of the instances in the project.